src/components/uploader/js/Uploader.js

```diff
@@ -139,18 +139,9 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
     };
 
     fluid.uploader.renderUploadTotalMessage = function (that) {
-        // Preservered for backwards compatibility, should be refactored post v1.5
-        var numReadyFiles = that.queue.getReadyFiles().length;
-        var bytesReadyFiles = that.queue.sizeOfReadyFiles();
-        var fileLabelStr = fluid.uploader.fileOrFiles(that, numReadyFiles);
-
         var totalCount = that.queue.files.length;
         var noFilesMsg = that.options.strings.progress.noFiles;
-
         var totalStateStr = fluid.stringTemplate(that.options.strings.progress.toUploadLabel, {
```

Comment:
Please verify if this string definition of progress.toUploadLabel is still needed: https://github.com/fluid-project/infusion/blob/master/src/components/uploader/js/FileQueueView.js#L397. Thanks.